installation/install.sh

```
@@ -3253,6 +3262,45 @@ is_install_vops(){
     pass
 }
 
+is_install_fluentbit_server(){
+    echo_subtitle "Install fluentbit server"
+    if [ "$BASEARCH" = "x86_64" ]; then
+        if [ "$ZSTACK_RELEASE" = "h84r" ] || [ "$ZSTACK_RELEASE" = "uos20r" ]; then
+            echo "Installing libpq for fluent-bit..." >> $ZSTACK_INSTALL_LOG
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)